docs/faq.rst

```diff
@@ -187,6 +187,38 @@ all the part files and manually concatenate them together.
 
 For more details, see :ref:`checkpoints_parts`.
 
+My repository is corrupt, how can I restore from an older but working copy?
+---------------------------------------------------------------------------
+
+The recommended method is to delete the corrupted repository, and then copy the
+working repo to the same location. The delete command will completely remove the
+corrupt repo and delete the corresponding cache and security subdirectory in
+``~/.config/borg/security``, including the nonce value.
+
+If you use the repo copy for creating new archives, Borg will initialize a fresh
+nonce which is bad for security reasons (nonce values should never be reused).
+To prevent this, the security subdirectory should be saved before deleting, and
+later moved back into place.
+
+Example:
+
+::
+
+    # Get the repo ID from repo config.
+    REPO_ID=$(grep 'id = ' /path/to/repo-good/config | cut -d ' ' -f 3)
+
+    # Rename the repo security dir so Borg won't delete it.
+    cd ~/.config/borg/security
+    mv $REPO_ID $REPO_ID.backup
+
+    # Now delete and rename the security dir back.
+    borg delete /path/to/repo
+    mv $REPO_ID.backup $REPO_ID
+
+    # Finally copy the good repo to the original place.
+    cp -avr /path/to/repo-good /path/to/repo
```

Comment:
`rsync -avH` is a better tool here. it's better than cp regarding copying as much metadata as possible and also can work remotely. also if it gets interrupted, it does not start from the beginning again.

Reply:
Good point, I'll use that.